doc/whats_new/v0.21.rst

```diff
@@ -233,6 +233,13 @@ Support for Python 3.4 and below has been officially dropped.
   classes priors instead of the log of the priors. :issue:`12983` by
   :user:`Nicolas Hug<NicolasHug>`.
 
+- |Fix| ``feature_importances_`` in
+  :class:`ensemble.GradientBoostingClassifier` and
+  :class:`ensemble.GradientBoostingRegressor` now ignores the single
+  node trees in feature importance calculation, and returns an
+  array of zeros if all trees have only one single node (i.e. the root node).
+  :issue:`13620` by `Adrin Jalali`_.
```

Comment:
@NicolasHug I'm not sure how you'd like to merge the two changeslogs here.

Reply:
From the other PR:

```
  - sum up to ``1``
  - all the single node trees in feature importance calculation are ignored
  - in case all trees have only one single node (i.e. a root node),
    feature importances will be an array of all zeros.
```

All these cases apply here it seems?

Reply:
Not the first one, but merged them anyway.